vizro-core/src/vizro/_vizro.py

```diff
@@ -144,17 +144,9 @@ def _pre_build():
         # changes size.
         # Any models that are created during the pre-build process *will not* themselves have pre_build run on them.
         # In future may add a second pre_build loop after the first one.
-
```

Comment:
Do all of these py. files need to change? I am not sure if something went wrong with the merge on main, but I would have expected only md files to change

Reply:
@petar-qb - I think these changes come from the dynamic filter PR and shouldn't be here right?

Reply:
Great catch @huong-li-nguyen!! 

There are probably more "conflicts" there.. so I'll review this PR today afternoon

Reply:
If this PR was not supposed to change any `py` files but only `md` files, the easiest way is probably just to copy paste the `src` folder from main into this branch to revert all changes or create a new PR 👍

Reply:
I agree with what you said. If you need any help I can jump in.